wagtail/admin/tests/test_account_management.py

```diff
@@ -332,6 +335,10 @@ def test_with_email_management_disabled(self):
         self.assertTemplateUsed(response, "wagtailadmin/account/account.html")
         self.assertNotContains(response, "id_name_email-email")
 
+        # Check if the default title does not exist, Check alternative title "Name"
```

Comment:
```suggestion
        # Check if the default title does not exist, Check alternative title "Name" based on settings 
```

Maybe a bit more explanation here could help.